CMake/resolve_dependency_modules/google_cloud_cpp_storage.cmake

```diff
@@ -13,11 +13,8 @@
 # limitations under the License.
 include_guard(GLOBAL)
 
-set_source(absl)
-resolve_dependency(absl 20240116 EXACT)
-
 set_source(gRPC)
-resolve_dependency(gRPC 1.48.1 EXACT)
+resolve_dependency(gRPC CONFIG REQUIRED)
```

Comment:
You don't have to enforce `CONFIG` mode as it's the default, it will only look for a `Find*.cmake` if no config is found. 

If there is no specific reason why this is necessary I would prefer to keep the option open for people to supply find modules if necessary for what ever reason (otherwise they would need to patch this out... like we have done for e.g. folly^^). This goes for the other uses of `CONFIG` as well of course.

Reply:
Also dropping the minimum?

Reply:
Without the CONFIG, I was getting the weird errors about missing absl symbols.

Reply:
See my comment about protobuf.

Reply:
Should we keep the minimum version here as well?

Reply:
I just put back the minimum version, to advertise the version that we know that works. Thanks.